src/uiScreens.c

```diff
@@ -21,6 +21,60 @@ void ui_displayPathScreen(
 	);
 }
 
+// the given path typically corresponds to an account
+// if it contains anything more, we display just the whole path
```

Comment:
I would expect a condition checking the path length in the code below to leave out printing of the account number, but I don't see anything like that. How/where do you actually check it?

Reply:
I added the missing condition.